src/app/components/Tabs/Tabs.module.css

```diff
@@ -0,0 +1,12 @@
+.tabs {
+  display: grid;
+  grid-auto-flow: column;
+  background-color: var(--main-color);
+  opacity: 45%;
+  padding: 1em;
+}
+
+.tabs__active {
+  background-color: #fff;
+  opacity: 100%;
```

Comment:
This is always 100%. Maybe you mixed opacity of the container and links?
As an alternative, change the color.